test/test.sh

```diff
@@ -6,6 +6,6 @@ cd test
 rm -rf temporal-polyfill
 git clone --recurse-submodules https://github.com/js-temporal/temporal-polyfill.git
 cd temporal-polyfill
-npm i
+npm i --ignore-scripts
 TEST262=1 npm run build
 node --loader ../resolve-test.mjs ../runtest262.mjs
```

Comment:
@12wrigja Instead of adding complexity to temporal-polyfill by splitting up into two files in that repo, could we simplify both repos by changing this line to directly run runtest262.mjs in the cloned proposal-temporal repo? Then we could delete the runtest262.mjs copy from this repo and remove the need to change proposal-temporal. 

Like this:

```suggestion
node --loader ../resolve-test.mjs ./runtest262.mjs
```

Reply:
erk, I meant "js-temporal/temporal-polyfill" instead of "proposal-temporal" above.

Reply:
I think that might be possible using `npm link`, but by default if we changed nothing that would clone and run the latest current version of the runner specified within our package.json (https://github.com/js-temporal/temporal-polyfill/blob/445bfd3d4efabec2d23372c9cc92f8d8619c822e/package.json#L91), not the version in this repo.

Let me play around with that today and see if I can adjust this script to do that.

Reply:
Done - this nicely cleans up the testing story for this repo.

Reply:
Nice! I assume that `npm link` is a cleaner way of doing the same thing that the (now deleted) `resolve-test.mjs` was doing?